python/ray/tune/result.py

```diff
@@ -116,13 +116,22 @@
 STDOUT_FILE = "__stdout_file__"
 STDERR_FILE = "__stderr_file__"
 
+
+def _get_defaults_results_dir() -> str:
+    return (
+        # This can be overwritten by our libraries
+        os.environ.get("RAY_AIR_LOCAL_CACHE_DIR")
+        # This is a directory provided by Bazel automatically
+        or os.environ.get("TEST_TMPDIR")
+        # This is the old way to specify the results dir
+        or os.environ.get("TUNE_RESULT_DIR")
```

Comment:
should this be deprecated in the future in light of this new API?

Reply:
Yes! Updated